README.md

```diff
@@ -21,3 +21,9 @@ We love contributors! Please see our [Contributing Guide][] for ways you can hel
 [Contributing Guide]: CONTRIBUTING.md
 [spreadsheet maintained by IQSS]:https://docs.google.com/spreadsheets/d/1l2R9D1FQy88qVzg2bI6L1LgplmM2l7pnMI80jdiz4fk/edit?usp=sharing
 [crowdsourced spreadsheet]: https://docs.google.com/spreadsheets/d/1bfsw7gnHlHerLXuk7YprUT68liHfcaMxs1rFciA-mEo/edit#gid=0
+
+## with docker
+
+`docker-compose up -d ` and go to [http://127.0.0.1/](http://127.0.0.1/).
```

Comment:
It would be cool to just use `docker run`, so people don't need to install `docker-compose`, too.
See below, too.

Reply:
```
docker run -v "$(pwd)":"$(pwd)" -v /var/run/docker.sock:/var/run/docker.sock  --workdir="$(pwd)" -ti --rm dduportal/docker-compose:latest up
```
:wink: